### PR TITLE
Cockpit 224

### PR DESCRIPTION
**Pull request by KKoukiou**
Preview: https://kkoukiou.github.io/cockpit-project.github.io/blog/cockpit-224.html

### CODE REVIEW

File: _posts/2020-07-22-cockpit-224.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Cockpit 224
```
Comment (martinpitt) [inline _posts/2020-07-22-cockpit-224.md:2]: How about devoting this blog post entirely to cockpit-podman 20, and drop the header prefixes? Or at least mention cockpit-podman in title:?

File: _posts/2020-07-22-cockpit-224.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+category: release
+summary: Cockpit Podman new features and Cockpit bug fixes
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 224.
+
+### Podman: Support checkpoint and restore operations for containers
+
+Checkpointing a container stops the container while writing the state of all processes in the container to disk.
```
Comment (martinpitt) [inline _posts/2020-07-22-cockpit-224.md:16]:
```suggestion
Checkpointing a container stops the container and writes the state of all container processes to disk.
```

File: _posts/2020-07-22-cockpit-224.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+With this a container can later be restored and continue running at exactly the same point in time as the checkpoint.
```
Comment (martinpitt) [inline _posts/2020-07-22-cockpit-224.md:17]:
```suggestion
Such a checkpoint can be restored later to resume the container in the exact state as it was at checkpoint time.
```

File: _posts/2020-07-22-cockpit-224.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+This capability requires [CRIU 3.11](https://www.criu.org/Main_Page) or later installed on the system and currently
```
Comment (martinpitt) [inline _posts/2020-07-22-cockpit-224.md:18]:
```suggestion
This capability requires [CRIU 3.11](https://www.criu.org/Main_Page) or later installed. Currently
```

File: _posts/2020-07-22-cockpit-224.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+only runc container runtime which is available on cgroupv1 systems is supported.
```
Comment (martinpitt) [inline _posts/2020-07-22-cockpit-224.md:19]:
```suggestion
only the `runc` container runtime is supported; this is only available with `cgroupv1` systems.
```

File: _posts/2020-07-22-cockpit-224.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+Container details now contain basic networking information.
+
+![Cockpit podman networking info](/images/podman-networking-info.png){:.screenshot}
+
+Thanks to Eric Jeffrey for this contribution!
+
+### Podman: Enable filtering images and containers by owner
+
+If user wants to work only with system or rootless containers now they can switch between these two modes.
```
Comment (martinpitt) [inline _posts/2020-07-22-cockpit-224.md:33]:
```suggestion
If user wants to work only with system or their own user containers, they can filter these now.
```

File: _posts/2020-07-22-cockpit-224.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+![Cockpit podman owner filter](/images/podman-owner-filter.png){:.screenshot}
+
+### Podman: Don't show intermediate images by default
```
Comment (martinpitt) [inline _posts/2020-07-22-cockpit-224.md:37]: I'm fairly sure we never showed intermediate images.

```suggestion
### Podman: Optionally show intermediate images
```

File: _posts/2020-07-22-cockpit-224.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+Intermediate images are not shown by default but are hidden and user can opt-in to see them.
```
Comment (martinpitt) [inline _posts/2020-07-22-cockpit-224.md:39]:
```suggestion
Intermediate images are hidden by default, but there is a new control to show them.
```

File: _posts/2020-07-22-cockpit-224.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+![Cockpit podman intermediate images](/images/podman-intermediate-images.png){:.screenshot}
+
+### Podman : Enable setting up SELinux label when mounting volumes
+
+When creating new container and mounting volumes :z or :Z labels can be specified.
```
Comment (martinpitt) [inline _posts/2020-07-22-cockpit-224.md:45]:
```suggestion
When creating new container and mounting volumes, `:z` or `:Z` labels can be specified.
```

File: _posts/2020-07-22-cockpit-224.md (hunk 2 above)
Comment (KKoukiou) [reply _posts/2020-07-22-cockpit-224.md]: Decided to not include this commit in this release.

File: _posts/2020-07-22-cockpit-224.md (hunk 1 above)
Comment (garrett) [reply _posts/2020-07-22-cockpit-224.md:2]: Yeah, makes sense!

**Comment by martinpitt** [comment]
So what's left is the updated title and adding the bodhi URLs. release is done.

**Pull request merged**